duckduckgo/duckduckgo-privacy-extension
# Add anonymous engagement metrics

**Reviewer:** @andrey-p 



## Description:
Setting up anonymous engagement metrics for the extension.
Learn more here: https://duck.co/help/privacy/atb


## Steps to test this PR:
1. Build and reload extension
2. Navigate through sites, open the popup, look at the subviews etc
3. Inspect the Network requests in the background, make sure the right metrics are sent as per specs
4. There are also new unit tests

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
